### DISCUSSION TITLE
Virtual scroll can be supported in QTree

### DISCUSSION THREAD

**Discussion by wb-wenbei** [quasarframework/quasar#14006] [Ideas / Proposals]
To many nodes in QTree leads to rendering jam.

**Comment by MilosPaunovic** [comment]
Why not utilize lazy loading?
[https://quasar.dev/vue-components/tree#lazy-loading](https://quasar.dev/vue-components/tree#lazy-loading)

**Reply by wb-wenbei** [reply]
That is a good way, but it can't cover all scenes.
It can't alleviate the problem when too many nodes in the same children.

**Reply by metalsadman** [reply]
try freezing the data.

**Reply by wb-wenbei** [reply]
The root of the problem is dom rendering.
Too many doms need to render on the page.
